MaterialProgressBar/src/main/ets/components/MaterialProgressBar.ets

```diff
@@ -23,22 +23,12 @@ struct MaterialProgressBar {
   modelBuild: any
 
   aboutToAppear() {
-    console.log('style:' + this.mStyle)
-    console.log('modelBuild:' + JSON.stringify(this.modelBuild))
     this.modelBuild.style = this.mStyle
   }
   onPageShow() {}
 
   build() {
-    if (this.modelBuild) {
-      if (this.mStyle == MaterialProgressStyle.MaterialCircular) {
         MaterialComponent(this.modelBuild)
```

Comment:
Add a check for mStyle if its null or undefined..If true, then set the progress style to LINEAR and execute the code that follows below it

Reply:
Done